### PR TITLE
Healthchecks

### PR DESCRIPTION
**Pull request by Nesze**
First draft of healthchecks. Consuming from Kafka is not yet implemented.

Note: Also, feel free to leave comments on any lines, not just related to this pull requests.

### CODE REVIEW

File: kafka-bridge.go [hunk 1]
```diff
@@ -214,9 +190,13 @@ func (bridge BridgeConfig) forwardMsg(kafkaMsg string) {
 	resp, err := client.Do(req)
 	if err != nil {
 		fmt.Printf("Error: %v\n", err.Error())
-		return
+		return err
 	}
 	fmt.Printf("\nResponse: %+v\n", resp)
+	if resp.StatusCode != http.StatusOK {
+		return errors.New("Forwarding message is not successful")
```
Comment (Zapadlo) [inline kafka-bridge.go]: Include the `resp.StatusCode`?

File: kafka-bridge.properties [hunk 2]
```diff
@@ -10,7 +10,7 @@ zookeeper_connect=ftper05715-lvpr-uk-int:2181,ftper05716-lvpr-uk-int:2181,ftper0
 zookeeper_timeout=10s
 
 #HTTP endpoint
-http_endpoint=http://localhost:8080/notify
+http_endpoint=http://localhost:8081/notify
```
Comment (Zapadlo) [inline kafka-bridge.properties]: Whats this? Cadvisor is running globally on 8081

Comment (Nesze) [inline kafka-bridge.properties]: the bridge runs on UCS machines

**Comment by Zapadlo** [comment]
+1

**Pull request merged**